front/app/containers/Admin/projects/project/events/components/DateTimeSelection/TimeInput.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, useLayoutEffect, useRef } from 'react';
+
+import {
+  Tooltip,
+  Button,
+  Box,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import { TIMES } from './constants';
+
+const BUTTON_HEIGHT = 45;
+
+interface Props {
+  selectedDate: Date;
+  onChange: (newDate: Date) => void;
+}
+
+const TimeInput = ({ selectedDate, onChange }: Props) => {
```

Comment:
I understand your point, but the HTML `select` is too limited in terms of what styles you can apply. For the same reason, libraries like `react-select` don't use it either. And since a11y is indeed not an issue for the admin screen I will skip the keyboard navigation issues for now (except for the escape one, that one is really strange)